object-handlers.go

```diff
@@ -946,21 +946,40 @@ func (api objectAPIHandlers) CompleteMultipartUploadHandler(w http.ResponseWrite
 		completeParts = append(completeParts, part)
 	}
 	// Complete multipart upload.
+	// Send 200 OK
+	setCommonHeaders(w)
+	w.WriteHeader(http.StatusOK)
```

Comment:
@harshavardhana you are right, I shouldn't be using `writeErrorResponse` in the `if-err-not-nil-block`. As per AWS docs, it is OK to send `200 OK` and send error as part of the body.